src/views/Dashboard.vue

```diff
@@ -21,32 +21,40 @@
   -->
 
 <template>
-	<DashboardWidget :items="cards"
-		empty-content-icon="icon-deck"
-		:empty-content-message="t('deck', 'No upcoming cards')"
-		:show-more-text="t('deck', 'upcoming cards')"
-		:show-more-url="showMoreUrl"
-		:loading="loading"
-		@hide="() => {}"
-		@markDone="() => {}">
-		<template v-slot:default="{ item }">
-			<a :key="item.id"
-				:href="cardLink(item)"
-				target="_blank"
-				class="card">
-				<div class="card--header">
-					<DueDate class="right" :card="item" />
-					<span class="title">{{ item.title }}</span>
-				</div>
-				<ul v-if="item.labels && item.labels.length"
-					class="labels">
-					<li v-for="label in item.labels" :key="label.id" :style="labelStyle(label)">
-						<span>{{ label.title }}</span>
-					</li>
-				</ul>
-			</a>
-		</template>
-	</DashboardWidget>
+	<div>
+		<DashboardWidget :items="cards"
+			empty-content-icon="icon-deck"
+			:empty-content-message="t('deck', 'No upcoming cards')"
+			:show-more-text="t('deck', 'upcoming cards')"
+			:show-more-url="showMoreUrl"
+			:loading="loading"
+			@hide="() => {}"
+			@markDone="() => {}">
+			<template v-slot:default="{ item }">
+				<a :key="item.id"
+					:href="cardLink(item)"
+					target="_blank"
+					class="card">
+					<div class="card--header">
+						<DueDate class="right" :card="item" />
+						<span class="title">{{ item.title }}</span>
+					</div>
+					<ul v-if="item.labels && item.labels.length"
+						class="labels">
+						<li v-for="label in item.labels" :key="label.id" :style="labelStyle(label)">
+							<span>{{ label.title }}</span>
+						</li>
+					</ul>
+				</a>
+			</template>
+		</DashboardWidget>
+		<div class="center-button">
```

Comment:
The issue with this is that the DashboardWidget is supposed to be always the same height, which is 7 items so with 7 showing this will exceed the size that is is supposed to have.

I remember we had an issue in server about allowing to add a create button to the top header, but I cannot find it at the moment. 

So for a possible solution in the meantime I think we should rather limit the the listed cards then and manually add a show more action and the button as you do it right now.

https://github.com/nextcloud/nextcloud-vue-dashboard/blob/master/src/DashboardWidget.vue#L70-L76